scripts/source/prepare_ci_kafka.sh

```diff
@@ -62,7 +62,7 @@ for filename in $kafka_data_files; do
     echo "Fulfill kafka topic $topic with data from $base"
     # binary data, one message a file, filename/topic ends with "bin"
     if [[ "$topic" = *bin ]]; then
-        ${KCAT_BIN} -P -b 127.0.0.1:29092 -t "$topic" "$filename"
+        ${KCAT_BIN} -P -b 127.0.0.1:29092 -K ^ -t "$topic" "$filename"
```

Comment:
any hint on this?

Reply:
From the `kafkacat` command help.
```
  -K <delim>         Print message keys prefixing the message
                     with specified delimiter.
 ```
 
And it will have an impact as a producer.

Reply:
Also adjusted in simulation test. d491231